### PR TITLE
Issue-638 Expose version and git hash for all packages

### PR DESCRIPTION
**Pull request by kjaiswal**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: packages/taquito/version-stamping.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const { resolve, relative } = require('path');
+const { writeFileSync } = require('fs-extra');
+const { version } = require('./package.json');
```
Comment (jevonearth) [inline packages/taquito/version-stamping.js:3]: Won't this cause getVersion to show the version tag of the last released version all the time? If we are building a preview build from CI, having the branch name in the `VersionInfo.version` property would be reasonable, and that would clearly signal if the build is a preview, or an official build.

File: packages/taquito-beacon-wallet/package.json [hunk 2]
```diff
@@ -30,8 +30,9 @@
     "lint": "tslint  --project tsconfig.json -t codeFrame 'src/**/*.ts' 'test/**/*.ts'",
     "precommit": "lint-staged",
     "prebuild": "rimraf dist",
-    "build": "tsc --project ./tsconfig.prod.json --module commonjs && rollup -c rollup.config.ts ",
-    "start": "rollup -c rollup.config.ts -w"
+    "version-stamp": "node ../taquito/version-stamping.js",
+    "build": "npm run version-stamp && tsc --project ./tsconfig.prod.json --module commonjs && rollup -c rollup.config.ts ",
+    "start": "npm run version-stamp && rollup -c rollup.config.ts -w"
   },
```
Comment (AlirezaHaghshenas) [inline packages/taquito-beacon-wallet/package.json:36]: Is it a good idea to add a line: To all `package.json`s? This causes the file to be present right after `npm i`, probably avoiding any possible errors people might see before their first `npm start`

```
"postinstall": "npm run version-stamp"
```

File: packages/taquito/version-stamping.js (hunk 1 above)
Comment (AlirezaHaghshenas) [reply packages/taquito/version-stamping.js:3]: If I remember correctly, for the pre-release packages published from the CI, version will properly reflect the branch.
But exposing the branch name is a great idea anyway

File: packages/taquito-beacon-wallet/package.json (hunk 2 above)
Comment (kjaiswal) [reply packages/taquito-beacon-wallet/package.json:36]: This is a great recommendation, I have added this.

File: packages/taquito/version-stamping.js (hunk 1 above)
Comment (kjaiswal) [reply packages/taquito/version-stamping.js:3]: I checked on preview builds repository for this PR build it has right version in version.js file i.e. 8.1.0-94548f5e--781. I will check final build version and comment back.

```
exports.VERSION = {
    "commitHash": "94548f5e6b265091bf7c278ded9a866eb3f21152",
    "version": "8.1.0-94548f5e--781"
};

Comment (kjaiswal) [reply packages/taquito/version-stamping.js:3]: I created a dummy version "9.1.2" and followed steps for final release as per notion till part where we push to master/npm. 

I checked packages and all had right version.

```
export const VERSION = {
    "commitHash": "9b638f88e8ca6b2716c2a49462a525b35012804b",
    "version": "9.1.2"
};

**Review by jevonearth — APPROVED** [review]

**Pull request merged**